.github/bin/make-compilation-db.sh

```diff
@@ -16,10 +16,14 @@
 set -u
 set -e
 
-bazel fetch ...  # no double-slash: seems to problematic on Windows
-
 readonly OUTPUT_BASE="$(bazel info output_base)"
-python3 "${OUTPUT_BASE}/external/com_grail_bazel_compdb/generate.py"
+
+readonly COMPDB_SCRIPT="${OUTPUT_BASE}/external/com_grail_bazel_compdb/generate.py"
+if [ ! -r "${COMPDB_SCRIPT}" ]; then
```

Comment:
if you like a bit more compact form this could also be written:

```sh
[ -r "${COMPDB_SCRIPT}" ] || bazel fetch ...
```

It's fine as-is though.

Reply:
Nice, I like that idea. Done.